### DISCUSSION TITLE
print continuous chars with ascii index

### DISCUSSION THREAD

**Discussion by ruediheimlicher** [olikraus/u8g2#2493] [Q&A]
I want to print a char as result of a calculation. e.g ('A'+ i) with i in a range of 0 to 10
In C this is possible, but with a u8g2-call?

**Comment by olikraus** [comment]
you could use the drawGlyph function (https://github.com/olikraus/u8g2/wiki/u8g2reference#drawglyph).
The result of your calculation is a valid unicode, which can be passed to drawGlyph

**Comment by ruediheimlicher** [comment]
thanks, works.
Mit freundlichen Grüssen
Ruedi Heimlicher
